### DISCUSSION TITLE
Make visible the services launched in command line

### DISCUSSION THREAD

**Discussion by ddecrulle** [InseeFrLab/onyxia#450] [Ideas]
I think it would be interesting to make visible in the UI the services launched in command line. 

I think there are two advantages: 
- An educational benefit -> better understand what onyxia does when you launch a service.
- An advantage for integration tests -> to be able to test the integration of a new version of a chart helm

Of course I don't think about the integration of any chart but a chart known by onyxia (present in the different chart repositories)

**Comment by olevitt** [comment]
API-side we have this WIP PR that intends to return the manifest created and may also return the helm cli used and also supports propre dry-runs. This may be a good starting point

**Comment by garronej** [comment]
Yes! Agreed 100%, this is a priority
